### PR TITLE
update usb example info in README.md

### PR DESCRIPTION
**Pull request by kilograham**

### CODE REVIEW

**Review by aallan — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -176,18 +176,47 @@ App|Description
 
 ### USB Device
 
+#### TinyUSB Examples 
+
+All but one USB device examples come directly from the TinyUSB device examples directory [here](https://github.com/hathach/tinyusb/tree/master/examples/device).
```
Comment (lurch) [inline README.md:181]: Would "All but one of the USB device examples" be better grammatically?

**Comment by lurch** [comment]
LGTM otherwise :+1:

File: README.md (hunk 1 above)
Comment (kilograham) [reply README.md]: fixed

**Comment by lurch** [comment]
I guess this ought to be cherry-picked onto `develop` too?

**Pull request merged**